modules/sharedaddy/sharing.css

```diff
@@ -655,6 +655,7 @@ div.sharedaddy.sharedaddy-dark #sharing_email {
 	max-width: none;
 	padding: 1px 3px;
 }
+#jetpack-source_f_name{ display: none!important; position: absolute !important; left: -9000px; }
```

Comment:
Why are we compressing the css on this line in this version of the file, when none of the surrounding lines are?